cpp/benchmarks/CMakeLists.txt

```diff
@@ -202,3 +202,8 @@ ConfigureBench(STRINGS_BENCH
   string/substring_benchmark.cpp
   string/translate_benchmark.cpp
   string/url_decode_benchmark.cpp)
+
+###################################################################################################
+# - json benchmark -------------------------------------------------------------------
+ConfigureBench(JSON_BENCH
+  string/json_benchmark.cpp)
```

Comment:
Blank line missing

Reply:
I think that's the way it's supposed to be.  Github was giving me the little red circle when it was in there.  Previous version also has no space.